src/LoggingFormats.jl

```diff
@@ -70,6 +70,17 @@ end
 transform(::Type{String}, v) = string(v)
 transform(::Type{Any}, v) = v
 
+# Use key information, then lower to 2-arg transform
+function transform(::Type{T}, key, v) where {T}
+    key == :exception || return transform(T, v)
+    if v isa Tuple && length(v) == 2
```

Comment:
Maybe make sure that it is also an exception?
```suggestion
    if v isa Tuple && length(v) == 2 && v[1] isa Exception
```